.command.run: line 284: TZ: unbound variable
Traceback (most recent call last):
  File "/app/compute_metrics.py", line 230, in <module>
    main(args)
  File "/app/compute_metrics.py", line 195, in main
    print(f"INFO: Wrote metrics JSON â†’ {out_json_path.relative_to(Path.cwd())}")
  File "/usr/lib/python3.8/pathlib.py", line 908, in relative_to
    raise ValueError("{!r} does not start with {!r}"
ValueError: 'assessment_results.json' does not start with '/home/zohaib/Downloads/brain_cancer_new/classification_metrics/classification/work/cb/b1b7ac86aedfa58cb01e24e2e7aa6e'
